server/channels/db/migrations/postgres/000110_update_vacuuming.up.sql

```diff
@@ -0,0 +1,4 @@
+alter table posts set (autovacuum_vacuum_scale_factor = 0.1, autovacuum_analyze_scale_factor = 0.05);
```

Comment:
I'd almost expect `audits` to be here as I remember it being a particularly write intensive table, but maybe it's not that critical.

Reply:
It did not come up in the analysis. I think it's because it just inserts rows and does not update/delete them. Therefore, not many dead tuples are created.